packages/web-components/fast-foundation/src/select/stories/select.stories.ts

```diff
@@ -22,6 +23,7 @@ export default {
         disabled: false,
         multiple: false,
         open: false,
+        placeholder: undefined,
```

Comment:
Does the `placeholder` arg need to be declared here if it's present in the `argTypes`?

Reply:
It does not